source/agora/api/Validator.d

```diff
@@ -93,4 +94,16 @@ public interface API : agora.api.FullNode.API
     ***************************************************************************/
 
     public bool receiveEnvelope (SCPEnvelope envelope);
+
+    /***************************************************************************
+
+        Reveal a pre-image
+
+        Params:
+            preimage = a PreimageInfo object which contains a hash and round
+                number
+
+    ***************************************************************************/
+
+    public void sendPreimage (PreimageInfo preimage);
```

Comment:
Naming wise, we have `receiveEnvelope` but `sendPreimage`. I don't mind either way but we should be consistent.

Reply:
Do you mean changing the function name into a name like receivePreimage? If you do, changing the name is not a problem for me. I mean I can change the neme.